eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -370,6 +370,7 @@ stages:
         targetArchitecture: x64
         useDevVersions: true # Use dev versions for CI validation of the experience. If we decide to ship assets from this leg, then we should remove this option.
         runTests: false # Temporarily do not run tests. The nuget comparison fails for some non-obvious reason and needs further investigation. Mostly, I'm not sure why it ever passed. https://github.com/dotnet/sdk/issues/42920
+        sign: true
```

Comment:
What's the long-term strategy for this switch? I assume we won't control this per job in the future but per whole build?

Reply:
It will probably still be per job but default to true. We dont' need to sign everything, like PGO.